specification/specs/common.ts

```diff
@@ -52,7 +52,8 @@ type ScrollIds = string
 type CategoryId = string
 type ActionIds = string
 type Field = string
-type Fields = string
+/** Wildcard (*) pattern or array of patterns containing source fields to return */
```

Comment:
I would reword this:
```suggestion
/** Path to field or array of paths. Some API's support wildcards in the path to select multiple fields.  */
```